onboard/cv/main.cpp

```diff
@@ -125,13 +188,14 @@ int main() {
           arTags[0].id = -1;
         }
       } else { //one tag found
-        arTags[0].distance = depth_img.at<float>(tagPair.first.loc.y, tagPair.first.loc.x);
+        if(!isnan(depth_img.at<float>(tagPair.first.loc.y, tagPair.first.loc.x)))
```

Comment:
What happens if this value is `nan`? do we then never set a bearing/distance/id for tag 0? 

It's my understanding that even if at the center of the tag our depth happens to be `nan`, the depth could till be `non-nan` for other points in the tag. I think in this case where `depth_img` at the centerpoint of the tag is `nan`, we should probably search for some other points on the tag that may not be nan, instead of just scrapping tag 0's information altogether.